### ISSUE TITLE
[APKPure] Root install for XAPK not yet implemented

### ISSUE THREAD

**Issue by StormProductionsMusic** [rumboalla/apkupdater#452]
Hi,

I've just updated to the latest version that includes APKPure, however, whenever I try to update an app through it, it simply doesn't wanna do it, without displaying any reason.

Here's a video showcasing the bug:
https://filetransfer.io/data-package/a2GIrL9R

Thanks!

**Comment by rumboalla** [comment]
Can you provide logs for this?

**Comment by StormProductionsMusic** [comment]
Update: Seems like this happens only when root install is enabled. 

If you tell me how to provide logs, I could try to do that – sorry, I'm not that good at that :)

**Comment by rumboalla** [comment]
Root install for XAPK files is not implemented yet.

**Comment by StormProductionsMusic** [comment]
Got it, will keep an eye on this issue when it'll be implemented. Will keep root install off until then.

**Comment by bonbonboi** [comment]
Same issue here. If the app give an option to download the apk(s) to the external storage I can install it manually. Also it doesn't show that it's downloading something aka download progress bar.

**Comment by StormProductionsMusic** [comment]
Yes, a progress bar would be neat to be honest. I don't even know what is the size of the download, how much it'll take, etc.
